transform/utilities.py

```diff
@@ -0,0 +1,53 @@
+"""This module contains utility function for calculating the fuel consumption of an aircraft, calculating
+Haversine distances and finding nearest airports based on longitude and latitude."""
+from math import sin, cos, acos, pi
+from datetime import datetime
+
+
+def haversine_distance(lat1: float, lon1: float, lat2: float, lon2: float) -> float:
```

Comment:
```suggestion
def calculate_haversine_distance(lat1: float, lon1: float, lat2: float, lon2: float) -> float:
```
more descriptive function name